modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/GETTING_STARTED.markdown

```diff
@@ -0,0 +1,186 @@
+# Getting Started with Liferay Workspace
+
+Complete documentation for Liferay Workspace can be found 
+[here](https://learn.liferay.com/dxp/7.x/en/developing-applications/tooling/liferay-workspace.html).
+
+### Folder Structure
+```
+my-project
+├── configs
+│   ├── common
+│   ├── dev
+│   ├── docker
+│   ├── local
+│   ├── prod
+│   └── uat
+├── ext
+│   ├── core ext
+│   └── module exts
+├── modules
+│   ├── apis
+│   ├── services
+│   ├── java widgets
+│   └── js widgets
+├── themes
+│   └── js themes
+└── wars
+    ├── java ee widgets (spring mvc, jsf, etc)
+    └── java themes
+```
+
+### Running Liferay DXP locally
+```
+my-project $ gw initBundle
```

Comment:
maybe we should use `./gradlew` since not everyone will have `gw`.  If they use our SDK installers, we do install `gw` so it would work.

Reply:
let's switch it to `blade gw`